### PR TITLE
Trim down and fix blacklist for #commie-subs

### LINKED ISSUE #9: Refine Blacklist Logic
Blacklisted word checks are pretty rudimentary right now; the system should redesigned to be more flexible.

### PR DESCRIPTION
**Pull request by kunaldes**
Array.prototype.forEach is overrated.

Fixes #9 

I should probably add tests... but what do you think about the nested for loop? I think the "optimal" solution would be to make a combined regex that we can check against but fuck that.

### CODE REVIEW

File: src/commands/command.js [hunk 1]
```diff
@@ -27,22 +25,29 @@ export class Command {
...
+      for (let word of message.split(/\s+/)) {
+        if (word.toLowerCase() === badword) {
+          log(`Blacklist triggered on "${badword}"`);
+          return true;
+        }
       }
-    });
+    }
 
-    return triggered;
```
Comment (IanMitchell) [inline src/commands/command.js]: This line was my favorite :(

**Comment by IanMitchell** [comment]
I think it looks good. The only thing I'd change (and it's super nit-picky, so feel free to ignore it) is swapping the inner / outer for-loop. Maybe it's just how I think, but it feels easier to understand "for each word, check each blacklist word" rather than "for each black list word, check each word"

**Comment by kunaldes** [comment]
idk i like it better this way. that way both loops are roughly "for each word in the blacklist, see if it's in the output string.

**Comment by IanMitchell** [comment]
:+1: fair point

**Pull request merged**